test/e2e/.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: node_js
+node_js:
+- '9'
+sudo: required
+addons:
+  chrome: stable
+install:
+- cd $TRAVIS_BUILD_DIR/test/e2e
+- npm install
+- npm run linter
+- npm run compile
+before_script:
+- npm run grid > /dev/null 2>&1 &  
+- sleep 10
```

Comment:
Why do you need Travis to sleep?

Reply:
In order to give for selenium grid to start